### PR TITLE
JavaScript capstone project 

### PR DESCRIPTION
**Pull request by berylcl**
At this milestone, we :
- Use ES6 syntax.
-  Use ES6 modules.
- Use callbacks and promises.
-  Use webpack.
- Apply JavaScript best practices and language style guides in code.
- Use AAA pattern for unit tests.
- Write units tests.
- Follow Gitflow.
- Solve simple git conflicts.
- Send and receive data from an API.
- Use API documentation.
- Use JSON.
- Make JavaScript code asynchronous.
-   Perform a code review for a team member.

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi team @berylcl, @ybadr99  🙋‍♀️,  

While you made a great effort in this project, I can't review your code.

## Invalid Code Review Request
 
You have submitted a project that has a README that is not professional. Here is a key point to consider when documenting your README.

- All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present. Please add the `Future Features` section and make sure that it includes 1-3 potential features to be added. ✔️ 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "bund",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "start": "webpack serve --watch"
```
Comment (Yerimah) [inline package.json]: - [ ] Hey guys, you have done an amazing job on this project but there is an issue with running the test cases. You don't seem to have any scripts installed to run the test. I would suggest that you look into this and fix it. Also, include the instruction on how to run the test in your README as well.

**Review by Yerimah — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 

Hi @Team, 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4d634a5755566852566e493378656f516f6d2f67697068792e676966


## Highlights
- Meaningful commit messages 👍 
- No linter errors 👍 
- Descriptive PR title and summary 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lookingfor00 — APPROVED** [review]
# Approved 🟢
Hi @berylcl , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
allfather-approves-approved

## Highlights
- Project look awesome 🍏 🌷 
- Code is clean and well organized 🧼 💯 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@Trast00) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**